packages/next/src/server/app-render/create-component-tree.tsx

```diff
@@ -409,7 +392,35 @@ async function createComponentTreeInternal({
     workStore.rootParams = currentParams
   }
 
-  //
+  // Only render metadata on the actual SSR'd segment not the `default` segment,
+  // as it's used as a placeholder for navigation.
```

Comment:
could you clarify this a bit more? the `default` segment could be what's SSRed when you load a page that has no `page.tsx` in a particular slot, but has a `default.tsx`

Reply:
Ideally it shouldn't render.
When the page is missing it renders 404, when the default of slot is rendered, it shouldn't overide the page metadata. However I found this is not running as expected right now. Created another task to track this issue and we can fix separately, as it's same between this branch and `canary` branch. I can follow up later